sites/public/styles/overrides.scss

```diff
@@ -56,9 +56,41 @@
           border: none;
         }
 
+        .navbar-dropdown-container {
+          margin-left: -144px;
+          width: auto;
+          .navbar-dropdown {
+            .navbar-dropdown-item-container {
+              display: flex;
+            }
+            .navbar-dropdown-item {
+              @apply border-t;
+              @apply border-gray-450;
+              @apply flex;
+              @apply flex-1;
+              @apply items-center;
+              @apply normal-case;
+              @apply pt-3;
```

Comment:
I think all these p's can be combined into `p-3`. What's here is the equivalent of:
```
padding-top: 0.75rem;
padding-bottom: 0.75rem;
padding-left: 0.75rem;
padding-right: 0.75rem;
```
and p-3 is
```
padding: 0.75rem;
```

Reply:
Done.